### ISSUE TITLE
Unable to Access PersonaInlineNavigationState Properties

### ISSUE THREAD

**Issue by protocorn93** [persona-id/inquiry-ios-2#7]
@mobile-build-persona Hi 👋

I would like to push the `PersonaInlineViewController` instance onto my navigation stack.

I understand that there are two interfaces available for controlling the presentation in `PersonaInlineViewController`:

- `navigateBack`
- `presentCancelModal`

Additionally, there is the `InquiryInlineDelegate`, which provides updates on the navigation status of `PersonaInlineViewController`.

However, since I cannot access the properties of `PersonaInlineNavigationState`, I am unable to fully control the presentation of `PersonaInlineViewController`.

Here is my intended implementation:

If `backButtonEnabled` is true, I would like to set my left navigation bar button as a back button.
If `cancelButtonEnabled` is true, I would like to set my left navigation bar button as a close button.
Could you please adjust the access control of `PersonaInlineNavigationState` properties to make this possible?

Thank you!

P.S) Could you please update [intergration guide](https://docs.withpersona.com/docs/ios-sdk-v2-integration-guide) about inline things on your website?

**Comment by jacoblange-dev** [comment]
Thanks for flagging, the properties on `PersonaInlineNavigationState` were intended to be public. I've just pushed the following changeset can release it by tomorrow.

``` /// A simple data type that represents the current enabled navigation actions in the persona flow
 public struct PersonaInlineNavigationState {
     /// If the current step has back navigation button enabled/shown as configured in the Inquiry Template
-    let backButtonEnabled: Bool
+    public let backButtonEnabled: Bool
     /// If the current step has cancel navigation button enabled/shown as configured in the Inquiry Template
-    let cancelButtonEnabled: Bool
+    public let cancelButtonEnabled: Bool
 }
 ```

> P.S) Could you please update [integration guide](https://docs.withpersona.com/docs/ios-sdk-v2-integration-guide) about inline things on your website?

Our inline support is bleeding edge and should be considered experimental, meaning the API might be unstable while we determine the correct interface. For this reason, we haven't pushed documentation publicly yet.

**Comment by protocorn93** [comment]
Thanks! 

> Our inline support is bleeding edge and should be considered experimental, meaning the API might be unstable while we determine the correct interface. For this reason, we haven't pushed documentation publicly yet.

Oh. I see. Thanks for your explanation

**Comment by jacoblange-dev** [comment]
Released as 2.23.3 https://github.com/persona-id/inquiry-ios-2/releases/tag/2.23.3
